helm/opendistro-es/files/configurer/index-templates/kubeaudit.template.json

```diff
@@ -0,0 +1,13 @@
+{
+  "index_patterns": [
+    "kubeaudit*"
+  ],
+  "template": {
+    "settings": {
+      "index.refresh_interval": "10s",
```

Comment:
What does this do? 10s sounds very frequent but maybe we need that?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-refresh-interval-setting
But yeah, this was added back in the day, and we can change it if we want to.

Reply:
Thanks! I think 10s makes sense with that explanation :slightly_smiling_face: